# Automatically Generated in 2017-06-12 22:33:47.452879
# MESSAGE: ODOMETER_ABS
# Description:Measurement of Odometry Sensor
uint8 ID = 136
uint8 sysid
uint8 compid
uint32 timestamp
float32 xdist
float32 ydist
float32 zdist
float32 xvelocity
float32 yvelocity
float32 zvelocity
int16 quality
